math/libqalculate/Portfile

```diff
@@ -29,6 +29,12 @@ checksums           rmd160  9da64547e5850c7ef9539d9a3a6c5816c355021b \
                     sha256  591598dedbcbd80119de052559873530030b3510bca2b0758f088cfb7dafb2ee \
                     size    2674322
 
+# https://github.com/Qalculate/libqalculate/issues/543
+# https://trac.macports.org/ticket/63645
+post-patch {
+    reinplace "s|\$(CXX_FOR_BUILD)|${configure.cxx}|" ${worksrcpath}/po-defs/Makefile.am
```

Comment:
Surely the authors of the build system did not intend you to patch this. There must be a way to set it from outside. Does setting the `CXX_FOR_BUILD` environment variable at configure time or perhaps at build time not work?

Reply:
I tried that, it did not. If you can suggest a working way, please.

Reply:
I can try replacing that with just `CXX`, that will perhaps allow Macports base to handle it correctly.